syntax = "proto3";
package logs;

service Logs {
    rpc getDockerLogs(Container) returns ( stream ContainerLogs) {}
    rpc getComposeLogs(Project) returns ( stream ContainerLogs) {}
}

message Container {
    string container_name = 1;
}

message Project {
    string project_name = 1;
}

message ContainerLogs {
    string message = 1;
}
